scene/main/scene_tree.h

```diff
@@ -88,7 +88,9 @@ class SceneTree : public MainLoop {
 
 	uint64_t tree_version = 1;
 	float physics_process_time = 1.0;
+	float physics_total_time = 1.0;
 	float process_time = 1.0;
+	float process_total_time = 1.0;
```

Comment:
Out of curiosity, why do process times start at `1.0` rather than `0.0`?

(Note that this isn't exclusive to your PR – see the unchanged lines.)

Reply:
Indeed I just copied the existing variables as is, which is obviously wrong in my case. The original `physics_process_time` and `process_total_time` are set each frame to the delta value so they are fine. However the ones I added are accumulated instead and thus must start in 0.
As for why it is like that, based on the log (originally defined in scene_tree.cpp), it has always been set to 1 since the initial commit Maybe some secret optimization technique?
Corrected and pushed.